samples/drivers/misc/grove_display/README.rst

```diff
@@ -46,5 +46,5 @@ described below:
    :goals: flash
    :compact:
 
-.. _Grove Base Shield: http://wiki.seeedstudio.com/wiki/Grove_-_Base_Shield
+.. _Grove Base Shield: https://wiki.seeedstudio.com/Base_Shield_V2/
```

Comment:
should we really link to successor?

Reply:
Given that the V2 of this board was released in Mar 2014 and this board was added to Zephyr in 2017, I assumed it was V2. Is that not the case?

Reply:
Good point, probably.